src/plugins/dashboard/public/dashboard_top_nav/dashboard_top_nav.tsx

```diff
@@ -172,15 +173,9 @@ export function DashboardTopNav({ embedSettings, redirectTo }: DashboardTopNavPr
     };
   }, [onAppLeave, getStateTransfer, hasUnsavedChanges, viewMode]);
 
-  const { viewModeTopNavConfig, editModeTopNavConfig } = useDashboardMenuItems({
-    redirectTo,
-    isLabsShown,
-    setIsLabsShown,
-  });
-
   const visibilityProps = useMemo(() => {
     const shouldShowNavBarComponent = (forceShow: boolean): boolean =>
-      (forceShow || isChromeVisible) && !fullScreenMode;
+      forceShow && isChromeVisible && !fullScreenMode;
```

Comment:
Change this to `&&` otherwise once `isChromeVisible` becomes true, the search bar is showed accordingly. But we don't need search bar in Security Solution dashboard.